pages/_app.page.tsx

```diff
@@ -189,7 +188,12 @@ const App = ({
                             pageContent
                           ) : (
                             <RouterGuard>
-                              <GraphQLProviders>{pageContent}</GraphQLProviders>
+                              <>
+                                <Helpjuice />
```

Comment:
Is there a reason why it's below the RouterGuard? Can users will access it on the Login page?

Reply:
This was because it is pulling user info from the session. But I should probably move it out of the `RouterGuard` and not send the user info if the session is missing.